neoscore/interface/path_interface.py

```diff
@@ -123,6 +128,7 @@ def _create_qt_object(self) -> QClippingPath:
             self.rotation,
             self.background_brush.qt_object if self.background_brush else None,
             defer_geometry_calculation=True,
+            # self.transform_origin,
```

Comment:
You just need to uncomment this line and bind it
```suggestion
            transform_origin=self.transform_origin,
```